paasta_tools/tron_tools.py

```diff
@@ -983,22 +1012,30 @@ def format_tron_action_dict(action_config: TronActionConfig):
                     "host_path": system_paasta_config.get_spark_kubeconfig(),
                     "mode": "RO",
                 },
-                # mount pod template file used for executors
-                {
-                    "container_path": spark_config.get(
-                        "spark.kubernetes.executor.podTemplateFile", ""
-                    ),
-                    "host_path": spark_config.get(
-                        "spark.kubernetes.executor.podTemplateFile", ""
-                    ),
-                    "mode": "RO",
-                },
+                # # mount pod template file used for executors
+                # {
+                #     "container_path": spark_config.get(
+                #         "spark.kubernetes.executor.podTemplateFile", ""
+                #     ),
+                #     "host_path": spark_config.get(
+                #         "spark.kubernetes.executor.podTemplateFile", ""
+                #     ),
+                #     "mode": "RO",
+                # },
```

Comment:
```suggestion
```
This can be removed, since we are using srv-configs for podTemplate for now and will implement in the spark-submit wrapper later.